sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.validateOffering.e2e-spec.ts

```diff
@@ -231,6 +201,440 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering",
       });
   });
 
+  it("Should validate an offering when valid data without study breaks is passed in for a full-time education program.", async () => {
```

Comment:
Why is the program intensity here an aspect of tests? I did not follow the reason.